cmdstanpy/utils.py

```diff
@@ -894,9 +894,12 @@ def scan_sampling_iters(
         data = line.split(',')
         if len(data) != num_cols:
             raise ValueError(
-                'line {}: bad draw, expecting {} items, found {}'.format(
+                'line {}: bad draw, expecting {} items, found {}\n'.format(
```

Comment:
we should add logic to calling function `scan_sampler_csv` so that it catches raised ValueError and adds filepath to the error message.

Reply:
Catching and raising from these errors causes all our tests for the exceptions to fail, as `assertRaisesRegex` only looks at the first exception, not any of the ones it is raised `from`. Still want me to do it?

Reply:
if there's a way to make the unit tests less brittle, that would be great, but yes, please make this change - I was tracking down this error with a user, and had the filepath name been available, that would have made things a whole lot easier.